src/main/scala/nl/knaw/dans/easy/multideposit/DDM.scala

```diff
@@ -75,9 +73,17 @@ object DDM {
       "DCX_RELATION_TITLE" -> "",
       "DCX_RELATION_LINK" -> "")
 
+  val composedTemporalFields: Dictionary =
```

Comment:
Because `DCT_TEMPORAL` and `DCT_TEMPORAL_SCHEME` come in pairs, I had to extract the former from the `metadataFields`.

Change in `DC_SUBJECT` and `DC_SUBJECT_SCHEME` is isomorphic.